test/optimizer/test_cascade_optimizer.py

```diff
@@ -91,6 +92,7 @@ def test_optimizer_post_process_strip_with_branch(self):
         self.assertTrue(isinstance(nest_plan.children[0], StoragePlan))
         self.assertTrue(isinstance(nest_plan.children[1], StoragePlan))
 
+    @pytest.mark.skip("No post-process tree")
     def test_optimizer_post_process_strip_without_branch(self):
```

Comment:
We can just remove these two test cases. I don't think we need them in the future also?